Makefile

```diff
@@ -129,9 +138,9 @@ clean-docker:
 	docker image prune -a
 	
 stop-docker:
-	@pkill -f "docker-compose .*stride.* logs" | true
-	@pkill -f "/bin/bash.*create_logs.sh" | true
-	@pkill -f "tail .*.log" | true
+	@-pkill -f "docker-compose .*stride.* logs" | true
```

Comment:
what was the reason for adding this? Isn't it the same as piping with `| true`? (except that it displays prints more logs)

Reply:
The piping didn't work for me (neither with `|| true`), this does - and it's the canonical way (unless there's something about the pipe that you really need?). If not, I can remove the `|true `.

Reply:
Gotchya - this must be a linux thing. Keeping the `-` is fine!